webtweets/js/tweets.js

```diff
@@ -1,79 +1,79 @@
+        window.setTimeout(parseFunc, 560);
+    }
 }
 
 function parser(data) {
-	var parsed = "";
-	var tweet = data.statuses[tweetNum].text;
-	var words = tweet.split(" ");
-	var loklakLinkCount = 0;
-	for (var word in words) {
-		if (words[word].startsWith("@")) {
-			parsed += "<a href='https://twitter.com/" + words[word].slice(1) + "' target='_blank'>" + words[word] + "</a> ";
-		} else if (words[word].startsWith("#")) {
-			parsed += "<a href='https://twitter.com/hashtag/" + words[word].slice(1) + "' target='_blank'>" + words[word] + "</a> ";
-		} else if (words[word].startsWith("http")) {
-			if (words[word].startsWith("http://loklak")) {
-				parsed += "<a href='" + data.statuses[tweetNum].links[loklakLinkCount] + "' target='_blank'>" + data.statuses[tweetNum].links[loklakLinkCount] + "</a> ";
-				loklakLinkCount += 1;
-			} else {
-				parsed += "<a href='" + words[word] + "' target='_blank' style='word-break:break-all'>" + words[word] + "</a> ";
-			}
-		} else {
-			parsed += words[word] + " ";
-		}
-	}
-	document.getElementById("tweet").innerHTML = parsed;
+    var parsed = "";
+    var tweet = data.statuses[tweetNum].text;
+    var words = tweet.split(" ");
+    var loklakLinkCount = 0;
+    for (var word in words) {
+        if (words[word].startsWith("@")) {
+            parsed += "<a href='https://twitter.com/" + words[word].slice(1) + "' target='_blank'>" + words[word] + "</a> ";
+        } else if (words[word].startsWith("#")) {
+            parsed += "<a href='https://twitter.com/hashtag/" + words[word].slice(1) + "' target='_blank'>" + words[word] + "</a> ";
+        } else if (words[word].startsWith("http")) {
+            if (words[word].startsWith("http://loklak")) {
+                parsed += "<a href='" + data.statuses[tweetNum].links[loklakLinkCount] + "' target='_blank'>" + data.statuses[tweetNum].links[loklakLinkCount] + "</a> ";
+                loklakLinkCount += 1;
+            } else {
+                parsed += "<a href='" + words[word] + "' target='_blank' style='word-break:break-all'>" + words[word] + "</a> ";
+            }
+        } else {
+            parsed += words[word] + " ";
+        }
+    }
+    document.getElementById("tweet").innerHTML = parsed;
 
-	var user = data.statuses[tweetNum].user;
-	var tweetDate = new Date(data.statuses[tweetNum].created_at);
-	document.getElementById("tweet-info").innerHTML = "Posted by <a href='https://twitter.com/" + user.screen_name + "'><b>" + user.name + "</b></a>, <a href='" + data.statuses[tweetNum].link + "'><i>" + tweetDate.toUTCString() + "</i></a>";
+    var user = data.statuses[tweetNum].user;
+    var tweetDate = new Date(data.statuses[tweetNum].created_at);
+    document.getElementById("tweet-info").innerHTML = "Posted by <a href='https://twitter.com/" + user.screen_name + "'><b>" + user.name + "</b></a>, <a href='" + data.statuses[tweetNum].link + "'><i>" + tweetDate.toUTCString() + "</i></a>";
```

Comment:
Line is longer than allowed. (241 > 129)

*LineLengthBear, severity NORMAL, section `default`.*